blocks/blockstore/blockstore.go

```diff
@@ -23,6 +23,7 @@ var log = logging.Logger("blockstore")
 var BlockPrefix = ds.NewKey("blocks")
 
 var ValueTypeMismatch = errors.New("The retrieved value is not a Block")
+var ErrHashMismatch = errors.New("Block on disk has different hash than expected.")
```

Comment:
- the block isn't neccessarily on disk, that's up to the datastore, isn't it? ;)
- IIRC no dot at the end. @RichardLitt dug up conventions for go error messages the other day